### ISSUE TITLE
iOS stopBackgroundService() called but red banner on home is still there

### ISSUE THREAD

**Issue by bobosette** [ekasetiawans/flutter_background_service#115]
Hi @ekasetiawans. Maybe you didn't see this comment so I open this issue cause I thing it's important to understand why this issue happens.
I don't know why when i call stopBackgroundService(), the red banner on ios home is still there. 
The app seems to reproduce the code inside stopBackgroundService() partially.
Do you have any idea?

`
FlutterBackgroundService().sendData(
{"action": "stopService"},
);

if (event["action"] == "stopService") {
TfliteAudio.stopAudioRecognition();
final json = RecognizerController.to.endReg();
FlutterBackgroundService().sendData(
{"isStopped": json},
);
await Future.delayed(const Duration(milliseconds: 1000), () {});
FlutterBackgroundService().stopBackgroundService();
}
`

**Comment by ekasetiawans** [comment]
@bobosette could you explain in more detail what the red banner is?

**Comment by bobosette** [comment]
like this:
![aXEkK](https://user-images.githubusercontent.com/13197553/158325682-80d90ccc-a808-4b98-abc5-60965bbc2035.png)

**Comment by ekasetiawans** [comment]
I never seen that banner before, is it really come from this plugin?

**Comment by bobosette** [comment]
from iOs 12 I think (I don't know exactly when they implemented that) when
an app is running a process in background, if you come back to the home,
this banner advices user that an app is running in background.

Il giorno mar 15 mar 2022 alle ore 08:29 Eka Setiawan Saputra <
***@***.***> ha scritto:

**Comment by ekasetiawans** [comment]
@bobosette the red banner is an indicator that the app is currently using microphone, so you should make sure your app has stopped using it.

https://stackoverflow.com/questions/13339078/red-banner-comes-and-disappear-suddenly-when-press-home-button-during-recording

**Comment by bobosette** [comment]
🤦🏻‍♂️ you're right. But now I have to understand why the microphone is still open

**Comment by bobosette** [comment]
Hi @ekasetiawans. I finally solved that issue. It was a problem between flutter_sound and flutter_background_service I think. 

Before this discussion, in my app I started the flutter_sound recorder first, and then I called the FlutterBackgroundService.onStart (inside the onStart() function there were some methods to analyze data independently of flutter sound). In this way i don't know why but flutter_sound crashed.

I changed the logic and I putted all the flutter_sound stuff inside the onStart() function.....and it worked fine!!!!
